IBAnimatableApp/IBAnimatableApp/Playground/Presentations/PresentingViewController.swift

```diff
@@ -45,7 +45,7 @@ final class PresentingViewController: AnimatableViewController, UIPickerViewData
   fileprivate let animations = ["None", "Flip", "CrossDissolve", "Cover(Left)", "Cover(Right)", "Cover(Top)", "Cover(Bottom)", "Zoom", "DropDown"]
   fileprivate let positions = ["Center", "TopCenter", "BottomCenter", "LeftCenter", "RightCenter"]
   fileprivate let sizes = ["Half", "Full", "custom(100)"]
-  fileprivate let keyboardTranslations = ["None", "MoveUp", "AboveKeyboard"]
+  fileprivate let keyboardTranslations = ["None", "MoveUp", "AboveKeyboard", "stickOrMoveUp"]
```

Comment:
Yeah I just checked out `IBEnum`. Not a big deal. I see more mismatching casing in the file. If anything this demonstrates that we accept both formats.